ThemeSetup.php

```diff
@@ -139,52 +147,6 @@ public static function init_gutenberg_config() {
 
 		// Default Gutenberg styles on Frontend.
 		add_theme_support( 'wp-block-styles' );
-
-		// Disable Font Sizes by default.
-		add_theme_support( 'editor-font-sizes' );
```

Comment:
Where did you placed this line? I couldn't find it in the other files.  This line is required in case the developer doesn't want to display the font size panel.

Reply:
this is placed in backend/WP/Gutenberg/Editor.php